### DISCUSSION TITLE
Documentation not updated

### DISCUSSION THREAD

**Discussion by sureshachary** [keycloak/keycloak#12835] [Archive]
The documentation for Keycloak Version 18.0.2 still refers to old documentation.
Only the version number has been updated.
Even the startup scripts and configurations have changed, still documentation refers to old version.
Current documentation says start using $ .../bin/standalone.sh but it has been changed to $ .../bin/kc.sh and this is not updated in documentation.

All the current documentation refers to old version. Please update.

**Comment by abstractj** [comment]
@sureshachary please create a GitHub issue reporting it as a bug https://github.com/keycloak/keycloak/issues and also add all the details like: the sections, URLs which are affected.
